code/datums/movement/movement.dm

```diff
@@ -45,11 +45,3 @@ var/const/MOVEMENT_STOP    = 0x0008
 		if((may_move & (MOVEMENT_PROCEED|MOVEMENT_HANDLED)) == (MOVEMENT_PROCEED|MOVEMENT_HANDLED))
 			return TRUE
 	return TRUE
-
-// Base
-/atom/movable/Destroy()
-	if(LAZYLEN(movement_handlers) && !ispath(movement_handlers[1]))
```

Comment:
why remove this?

Reply:
> why remove this?

https://github.com/DS-13-Dev-Team/DS13/blob/ff6aaafa5574132b7885cca6cc152f0119788779/code/game/atoms_movable.dm#L59 moved